process/analysis.py

```diff
@@ -7,6 +7,8 @@
 import yaml
 from subprocesses._utils import get_terminal_columns, print_autobreak
 
+import ee

```

Comment:
We need to think this through --- the earth engine workflow should only be optional, not required; earth engine should only be imported and related functions should only be run if the user has opted in to this.  

for now, perhaps include `import ee` within the `authenticate_gcloud_and_gee` function so that is only called if ` r.config['gee'] is True`?

Reply:
I suggested further below the following

> an efficient way to do this, since it will be checked at multiple points, could be to add a subfunction to ghsci object that 1) checks configuration of gee, and 2) tests import of some small but crucial library only present in the GEE-specific docker image (or some other simpler way you can think of that will test that GEE libraries are accessible for use).   That `ghsci.check_gee()` function could then be used any time a check is required if GEE analysis is to proceed.